google/api_core/grpc_helpers.py

```diff
@@ -79,7 +83,7 @@ def error_remapped_callable(*args, **kwargs):
     return error_remapped_callable
 
 
-class _StreamingResponseIterator(grpc.Call):
+class GrpcStream(grpc.Call, Generic[S]):
```

Comment:
Do you think this could be a breaking change? Could we keep the name as `_StreamingResponseIterator` ? There are many hits for `_StreamingResponseIterator` in Google search. I'm worried that changing `_StreamingResponseIterator` could cause issues downstream in user code as it is a response we provide.

Reply:
Makes sense, I'll create a type alias instead, like I did on the async side